tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -150,9 +144,9 @@ copp/test_copp.py::TestCOPP::test_trap_config_save_after_reboot:
 #######################################
 crm/test_crm.py::test_crm_fdb_entry:
   skip:
-    reason: "Unsupported topology, expected to run only on 'T0*' or 'M0' topology"
+    reason: "Unsupported topology, expected to run only on 'T0*' or 'M0/MX' topology"
     conditions:
-      - "'t0' not in topo_name and 'm0' not in topo_name"
+      - "'t0' not in topo_name and topo_name not in ['m0', 'mx']"
```

Comment:
We might need revisit this case. It says t0 is not support, but seems like t0-52 can be supported?